src/matcha_ml/infrastructure/modules/mlflow/mlflow_module/outputs.tf

```diff
@@ -0,0 +1,4 @@
+output "mlflow_tracking_url" {
+  description = "The tracking URL for MLFlow dashboard"
+  value = "http://${data.kubernetes_service.mlflow_tracking.status.0.load_balancer.0.ingress.0.ip}:${data.kubernetes_service.mlflow_tracking.spec.0.port.0.port}"
```

Comment:
Is it the same ingress that's used by seldon? If so, it won't be there, if seldon's not deployed. Potentially needs moving around